usaspending_api/api_contracts/contracts/v2/financial_balances/agencies.md

```diff
@@ -1,30 +1,28 @@
 FORMAT: 1A
 HOST: https://api.usaspending.gov
 
-# Agency Profile
+# Agency Profile [/api/v2/financial_balances/agencies/]
 
 These endpoints are used to power USAspending.gov's agency profile pages. This data can be used to better understand the different ways that a specific agency spends money.
 
-## Financial Balances [/api/v2/financial_balances/agencies/{?fiscal_year,funding_agency_id}]
+## Get Financial Balances [GET /api/v2/financial_balances/agencies/{?fiscal_year,funding_agency_id}]
 
 This endpoint returns aggregated balances for a specific government agency in a given fiscal year.
 
 + Parameters
-    + fiscal_year: 2017 (required, number)
+    + `fiscal_year`: `2017` (required, number)
```

Comment:
Do we need the tics if these are numbers?

Reply:
No, I can go through and remove the tics for everything that is not a string. I was going to go through and try to make the use of tics consistent but noticed that we used them differently all throughout the API contracts. Personally, think it helps readability to have them formatted as:

+ Parameters
   + name: `value` (required, type)
   + another_name (optional, type, nullable)

Reply:
As of the last meeting we've set standard for the keys but not the values, but keep in mind some of these pages are V1 endpoints that we aren't really maintaining anymore.

Reply:
Yeah.  Don't get me started on backticks.  Everyone seems to have a different opinion on how we should use them.  At a minimum, [they need to enclose attributes or values that contain keywords or special characters](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md#6-reserved-characters--keywords).

Reply:
This README doc needs work, but #5 and #6 mentions when to use backticks:
https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts